### PR TITLE
improve code using rubocop

### PR DESCRIPTION
**Pull request by pdkproitf**

### CODE REVIEW

File: app/views/books/show.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
       <div class="col-md-4">
         <div class="single-sidebar">
           <h2 class="sidebar-title">RELATED PRODUCTS</h2>
-          <% if !@book.nil? %>
+            <% unless @book.nil? %>
```
Comment (yoyocoder) [inline app/views/books/show.html.erb]: It looks like your indent is wrong?

File: spec/models/comment_spec.rb [hunk 2]
```diff
@@ -10,42 +10,45 @@
...
+      comment.date = nil
+    end
 
-    context "desc" do
+    context 'validete desc' do
       it { expect(comment.desc).not_to be_nil }
     end
 
-    context "user_id" do
+    context 'validete user_id' do
```
Comment (yoyocoder) [inline spec/models/comment_spec.rb]: validates

File: spec/models/comment_spec.rb [hunk 3]
```diff
@@ -10,42 +10,45 @@
...
+      it { expect(comment).to validate_numericality_of(:book_id)
+                                                .is_greater_than_or_equal_to(1)
+      }
     end
 
     context "user id" do
       it { expect(comment.user_id).to be_truthy }
-      it { expect(comment).to validate_numericality_of(:user_id).is_greater_than_or_equal_to(1) }
+      it { expect(comment).to validate_numericality_of(:user_id)
+                                              .is_greater_than_or_equal_to(1)
```
Comment (yoyocoder) [inline spec/models/comment_spec.rb]: don't need to enter the new line here, guys. `ugly format`

File: db/seeds.rb [hunk 4]
```diff
@@ -11,25 +13,24 @@
...
-    @publish = Publish.create(:name => publish.name)
-    10.times do
-      i = 0 if i >= @category.size
-      book = FactoryGirl.build(:book)
-      Book.create(
-                category_id: @category[i],
+  next unless Publish.find_by_name(publish.name)
+  @publish = Publish.create(name: publish.name)
+  10.times do
+    i = 0 if i >= @category.size
```
Comment (yoyocoder) [inline db/seeds.rb]: 10.times do |index|
// do something with index
end

File: db/seeds.rb [hunk 5]
```diff
@@ -11,25 +13,24 @@
...
-@category = Array.new
+@category = []
 
 12.times do
   category = FactoryGirl.build(:category)
-  if !Category.find_by_name(category.name)
-    @category.push(Category.create(:name => category.name).id)
+  unless Category.find_by_name(category.name)
+    @category.push(Category.create(name: category.name).id)
   end
```
Comment (yoyocoder) [inline db/seeds.rb]: you can use [`find_and_create_by`](http://apidock.com/rails/v4.0.2/ActiveRecord/Relation/find_or_create_by) to make it nicer

**Comment by yoyocoder** [comment]
@pdkproitf  What is `rubocop`?

File: spec/models/comment_spec.rb (hunk 3 above)
Comment (buiquangthang) [inline spec/models/comment_spec.rb]: he follow gem rubocop end edit it =)))

File: app/views/books/show.html.erb (hunk 1 above)
Comment (pdkproitf) [inline app/views/books/show.html.erb]: it still woking

**Comment by pdkproitf** [comment]
it is new style that you sended to me

File: app/views/books/show.html.erb (hunk 1 above)
Comment (pdkproitf) [inline app/views/books/show.html.erb]: got it

**Comment by yoyocoder** [comment]
👍

**Comment by yoyocoder** [comment]
You can merge it 💯

**Pull request merged**